.github/workflows/pep8Validation.yml

```diff
@@ -0,0 +1,13 @@
+on: [pull_request]
```

Comment:
Is it not possible to add this workflow as an additional step in our existing workflow ? CI would look like
1. Get latest code
2. Check pycodestyle
3. Validate Tests

Reply:
I thought it would be better to have unit tests GitHub action and formatting validation GitHub action in separate files. This way we have 2 checks, one for unit tests and second for PEP 8 formatting validation.

Reply:
We would still have two checks but part of same workflow. I have see this as common pattern in multiple repositories. If you think about it that's how the order of validation should be. 1. Validate Formatting 2. Validate tests.